packages/repository/git/src/git-repository-provider.ts

```diff
@@ -11,11 +11,15 @@ export type ParsedRemoteUrl = {
   toString: (format?: string) => string;
 };
 
-type GitRepositoryProviderOptions = Record<string, never>;
+type GitRepositoryProviderOptions = {
+  version?: string;
+};
```

Comment:
The version option should not be directly linked to repository, as we can have different identifier between manifest and repo. Example:

A monorepo of manifest having an independent versioning strategy.